lib/ansible/module_utils/basic.py

```diff
@@ -216,25 +216,32 @@
 _ANSIBLE_ARGS = None
 
 FILE_COMMON_ARGUMENTS = dict(
-    src=dict(),
+    # These are things we want. About setting metadata (mode, ownership, permissions in general) on
```

Comment:
Wonder if it's worth giving some details of how to know if something should be in here or not, ie what is "common"?
May help reduce confusion in the future and things like `backup`, `force` being added back in

Reply:
Yeah.  I can do that when we actually break basic.py apart.